crates/shielder_bindings/uniffi-bindgen.rs

```diff
@@ -0,0 +1,4 @@
+// binary for generating uniffi language bindings
```

Comment:
unless this is required for uniffi, I suggest moving this module to either `src/`, or better, to `src/bin/` - all other crates have all their source files under `src/`